src/fmu/dataio/_products/inplace_volumes.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, Final, List, Literal, Optional
+
+from pydantic import BaseModel, Field, RootModel
+from pydantic.json_schema import GenerateJsonSchema
+
+if TYPE_CHECKING:
+    from typing import Any, Mapping
+
+# These are to be used when creating the 'product' key in metadata.
+VERSION: Final[str] = "0.1.0"
+SCHEMA: Final[str] = (
+    "https://main-fmu-schemas-prod.radix.equinor.com/schemas"
+    f"/file_formats/volumes/{VERSION}/inplace_volumes.json"
+)
+
+
+class InplaceVolumesResultRow(BaseModel):
+    """Represents a row in a static inplace volumes export.
+
+    These fields are the current agreed upon standard result. Changes to this model
+    should increase the version number in a way that corresponds to the schema
+    versioning specification (i.e. they are a patch, minor, or major change)."""
+
+    FLUID: Literal["oil", "gas", "water"]
+    ZONE: str
+    REGION: str
+    FACIES: Optional[str] = Field(default=None)
+    LICENSE: Optional[str] = Field(default=None)
+
+    BULK: Optional[float] = Field(default=None, ge=0.0)
+    NET: Optional[float] = Field(default=None, ge=0.0)
+    PORV: Optional[float] = Field(default=None, ge=0.0)
```

Comment:
`BULK` and `PORV` could be set non optional. They will always have values since we merged #906 🙂 . 
If we choose to do #924 we can also do the same with `NET.` 
`HCPV` will still be None in water, so we will need to keep that optional even though it is required.